desktop/src/main/java/pmstation/integration/Mqtt.java

```diff
@@ -57,15 +53,36 @@ public void messageArrived(String topic, MqttMessage message) {
                 @Override
                 public void deliveryComplete(IMqttDeliveryToken token) {}
             });
+        } catch (MqttException e) {
+            logger.error("Unable to create a Mqtt Client", e);
+        }
+    }
+
+    private void connect() {
+        String username = Config.instance().to().getString(Config.Entry.MQTT_USERNAME.key());
```

Comment:
why not to create Client here?

Reply:
`connect` method moved to constructor. Reconnection is nost supported yet :)